src/Modules/StatCan.OrchardCore.Hackathon/Recipes/SolutionForm.recipe.json

```diff
@@ -0,0 +1,157 @@
+{
+    "name": "Hackathon.SolutionForm",
+    "displayName": "Solution Submission Form",
+    "description": "Creates a form that can be used by users to submit a solution",
+    "author": "StatCan Digital Innovation",
+    "website": "digital.statcan.gc.ca",
+    "version": "1.0.0",
+    "issetuprecipe": false,
+    "categories": [],
+    "tags": [],
+    "steps": [
+        {
+            "name": "Content",
+            "data":[
+                {
+                    "ContentItemId": "4fy9a011te6qx5611jv1hhtwwh",
+                    "ContentItemVersionId": "4h65th8hv448bzjc42z0myte51",
+                    "ContentType": "VueForm",
+                    "DisplayText": "Solution Submission",
+                    "Latest": true,
+                    "Published": true,
+                    "ModifiedUtc": "2021-08-17T15:00:04.285423Z",
+                    "PublishedUtc": "2021-08-17T15:00:04.301012Z",
+                    "CreatedUtc": "2021-08-16T17:04:30.808582Z",
+                    "Owner": "4p0a17tp39jfwsg1jghjrqqvw6",
+                    "Author": "test",
```

Comment:
Can you change owner an author for 
```
"Owner": "[js: parameters('AdminUserId')]",
"Author": "[js: parameters('AdminUsername')]"
```
This is to ensure that both the owner and author exist in the tenant that runs the recipe.